doc/usage/quickstart.rst

```diff
@@ -251,6 +251,13 @@ config values.
 
 .. todo:: Move this entire doc to a different section
 
+.. versionadded:: 4.0.0
+
+   It is now possible to use a :file:`sphinx.yaml`,
+   in-place of the :file:`conf.py` (the two may not co-exist).
```

Comment:
In my _basic_ English knowledge, I understand that both may or may not co-exist. So, in my head, this means that I can use `conf.py` and `sphinx.yaml` files together or only one of them. I'd change this sentence to "the two _must not_ co-exist" to avoid this confusion.

Reply:
done 👍